src/Dapr.Actors/ActorReference.cs

```diff
@@ -15,7 +15,7 @@ namespace Dapr.Actors
 {
     using System;
     using System.Runtime.Serialization;
-    using Dapr.Actors.Client;
+    using Client;
```

Comment:
I'm personally not a fan of abbreviating namespaces in this way, even if it's technically allowed.  For `Dapr.Client`, in particular, there's just too many other "client" types and I'd prefer to be explicit.  (If anything, I would propose moving `using` statements to be above the `namespace` declaration and use the `namespace <ns>;` form rather than `namespace <ns> { /* ... */ }`.  That's a significant refactor, however, so it's only been done on a piecemeal basis when working within individual files.)